doc/daal4py/make.bat

```diff
@@ -0,0 +1,36 @@
+@ECHO OFF
```

Comment:
is this script intended for manual docs generation ?

Reply:
yes, for building documentation on Windows